h2o-core/src/main/java/water/util/ArrayUtils.java

```diff
@@ -1600,4 +1600,14 @@ public static boolean equalsAny(long value, long...lhs) {
     return res;
   }
 
+  /**
+   * Convert an ArrayList of Integers to a primitive int[] array.
+   */
+  public static int[] unbox(ArrayList<Integer> arr) {
```

Comment:
A similar function in Apache Commons is called "toPrimitive", perhaps it might be better to adopt the same name.